src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const popTags = document.querySelectorAll('.population');
+const items = [...popTags].map(element => {
+  const text = element.innerText;
+
+  return parseFloat(text.replace(/,/g, ''));
+});
+
+const totalPop = items.reduce((sum, currentVal) => sum + currentVal, 0);
```

Comment:
```suggestion
const totalPopulation = items.reduce((sum, currentVal) => sum + currentVal, 0);
```
It is better to use full names